lua/nvim-treesitter/parsers.lua

```diff
@@ -18,6 +18,7 @@ local filetype_to_parsername = {
   pandoc = "markdown",
   rmd = "markdown",
   cs = "c_sharp",
+  blp = "blueprint",
```

Comment:
That's not how this is meant to be used: It maps _filetypes_ to parser names, not _extensions_.

You should add the mapping of `*.blp` to filetype `blueprint` in your config, then the `blueprint` parser will just work out of the box.


(The existing entries are misleading and partially wrong.)

Reply:
Fixed, thanks for the review